tests/test_orderbook/test_limit_orders.py

```
@@ -0,0 +1,85 @@
+        symbol=market_config.symbol,
+        account_id=maker_tester.account_id,
+        order_id=response.order_id,
+    )
+    assert cancel_response is not None
+
+    await maker_tester.wait.for_order_state(response.order_id, OrderStatus.CANCELLED)
+
+
+@pytest.mark.asyncio
+async def test_mass_cancel_clears_open_orders(
+    market_config: Union[SpotTestConfig, PerpTestConfig],
```

Ruff (0.15.12)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)